cpp/oneapi/dal/algo/decision_forest/backend/gpu/train_misc_structs.hpp

```diff
@@ -166,4 +169,5 @@ struct train_context {
     static constexpr inline Index preferable_sbg_size_ = 16;
     static constexpr inline Index max_local_block_count_ = 1024;
```

Comment:
Magic numbers?

Reply:
No, it is named constants, 
magic numbers - unique values with unexplained meaning or multiple occurrences which could (preferably) be replaced with named constants (https://en.wikipedia.org/wiki/Magic_number_(programming))